### ISSUE TITLE
Transpose z-index like a floors

### ISSUE THREAD

**Issue by Jabher** [HashanP/postcss-spiffing#2]
Map in accordance with http://en.wikipedia.org/wiki/Storey#Numbering

``` css
.a {z-index: ground;}
.b {z-index: 1;}
```

to

``` css
.a {z-index: 1;}
.b {z-index: 2;}
```

**Comment by hashanp** [comment]
Great idea! I don't want to confuse anyone too much though, so what I think I'll do is I'll have another property called `storey`. E.g.:

``` css
.a {storey:ground;}
.b {storey:1;}
```

is mapped to:

``` css
.a {z-index:1;}
.b {z-index:2;}
```

**Comment by igoradamenko** [comment]
@HashanP You can close it, 'cause it was implemented in #10 :3
